romancal/tweakreg/tests/test_tweakreg.py

```diff
@@ -627,96 +628,6 @@ def test_tweakreg_updates_group_id(tmp_path, base_image):
         res.shelve(model, 0, modify=False)
 
 
-@pytest.mark.parametrize(
-    "shift_1, shift_2, tolerance, is_small_correction",
-    [
-        (0, 0, 5, True),
-        (0, 3, 5, True),
-        (1, 1, 5, True),
-        (5, 5, (5**2 + 5**2) ** 0.5, True),
-        (5, 5, 5, False),
-        (5, 5, 1, False),
-        (5, 5, 3, False),
-        (5, 10, 5, False),
-    ],
-)
-def test_tweakreg_correction_magnitude(
-    shift_1, shift_2, tolerance, is_small_correction, request
-):
-    """
-    Test that TweakReg corrections are within tolerance.
-    All the parametrized values are in arcsec.
-    """
-    img1 = request.getfixturevalue("base_image")(shift_1=1000, shift_2=1000)
-    img2 = request.getfixturevalue("base_image")(
-        shift_1=1000 + shift_1 / 0.1, shift_2=1000 + shift_2 / 0.1
-    )
-    img1_wcs = copy.deepcopy(img1.meta.wcs)
-    img2_wcs = copy.deepcopy(img2.meta.wcs)
-
-    step = trs.TweakRegStep()
-    step.tolerance = tolerance / 10.0
-
-    assert step._is_wcs_correction_small(img1_wcs, img2_wcs) == is_small_correction
-
-
-@pytest.mark.parametrize(
-    "filename_list, expected_common_name",
-    (
-        (
-            [
-                "l1-sca1_cal.asdf",
-                "l1-sca2_cal.asdf",
-                "l1-sca3_cal.asdf",
-            ],
-            "l1-sca",
-        ),
-        (
-            [
-                "l1-270-66-gaia-2016-sca1_cal.asdf",
-                "l1-270-66-gaia-2016-sca2_cal.asdf",
-                "l1-270-66-gaia-2016-sca3_cal.asdf",
-            ],
-            "l1-270-66-gaia-2016-sca",
-        ),
-    ),
-)
-def test_tweakreg_common_name(filename_list, expected_common_name, request):
```

Comment:
I think this one is no longer needed since the library handles the grouping (there's no longer a need for `_common_name`). Is that right?

Reply:
Correct.